index.js

```diff
@@ -1,16 +1,34 @@
-const { question } = require("readline-sync");
-const { displayWordSoFar, isGameWon, isGameLost } = require("./gamelogic");
+const {question} = require("readline-sync");
+const {displayWordSoFar, isGameWon, isGameLost, showWrongLetterCount, showGallows} = require("./gamelogic");
 
 function game(word, guesses) {
-  console.log("Dit heb je tot nu toe geraden: ", guesses);
+    console.log("Dit heb je tot nu toe geraden: ", guesses);
+    console.log("Dit zijn de letters die je tot nu toe goed hebt: " + displayWordSoFar(word, guesses));
+    console.log("Aantal verkeerd geraden letters: " + showWrongLetterCount(word, guesses));
+    showGallows(word, guesses);
 
-  const letter = question("Raad een letter: ");
+    let letter = question("Raad een letter: ");
 
-  // voeg de geraden letter toe aan de array met guesses
-  guesses.push(letter);
+    if (letter.length > 1) {
```

Comment:
Interessante oplossing voor de dubbele letters. Rein had in zijn uitwerking een readline-sync method gebruikt, maar deze manier kan natuurlijk ook :)